use forked node-speaker implementation with warning log removed

AbdurrehmanSubhani opened the pull request:
This PR removes the buffer underflow warning originating from the node-speaker's coreaudio.c implementation.

- The update includes using our own forked repo with only the warning code line being commented out, the forked Repo is found here:
https://github.com/UpstreetAI/node-speaker


- Some package-lock updates are not directly associated with this update as they were integrated by running npm i before updating the speaker dep in package.json. (mostly ora package updates)

avaer commented:
Why not include the library into `packages` and use `file:` install?

avaer commented:
This is a good PR but we should probably ingest the change into the monorepo rather than depending on a `git` repo install (as far as I know we do not do this for any modules).

AbdurrehmanSubhani commented:
> This is a good PR but we should probably ingest the change into the monorepo rather than depending on a `git` repo install (as far as I know we do not do this for any modules).

- Added the suppressed speaker implementation to usdk/packages/speaker
- updated the package.json to use the local directory for installation

AbdurrehmanSubhani commented:
will have to fix build deps, moving to draft for now
